src/backend/marsha/core/api/account.py

```diff
@@ -1,15 +1,66 @@
 """Declare API endpoints for user and organisation with Django RestFramework viewsets."""
 from django.contrib.auth import get_user_model
 
-from rest_framework import viewsets
+from rest_framework import status, viewsets
 from rest_framework.decorators import action
+from rest_framework.generics import GenericAPIView
 from rest_framework.response import Response
+from rest_framework_simplejwt.exceptions import InvalidToken, TokenError
+from rest_framework_simplejwt.serializers import TokenVerifySerializer
+from rest_framework_simplejwt.settings import api_settings
+
+from marsha.core.simple_jwt.tokens import ChallengeToken, UserAccessToken
 
 from .. import permissions, serializers
 from ..models import Organization
 from .base import APIViewMixin, ObjectPkMixin
 
 
+class ChallengeTokenSerializer(TokenVerifySerializer):
```

Comment:
Should not be with the other serializers of the apllication in the `serializers` directory ?